ecs-deploy

```diff
@@ -349,7 +354,31 @@ else
           | grep -e "RUNNING") || :
 
         if [ "$RUNNING" ]; then
```

Comment:
In testing this I tried to update a service to use an invalid image name to simulate a failed deployment but it did not automatically revert. Troubleshooting a bit I believe the problem is here. Since line 352 is checking for tasks of the new task definition arn and a running status it never got to a running status because it could not pull the image. So then in line 356 when testing `if [ "$RUNNING" ]...` it was not true and did not enter the logic that would trigger a fallback. Perhaps in the use case you outlined though of failed migrations the task would be running for a period and this would catch it, but if it never gets to a running state (bad image, failed entrypoint/command, etc.) it doesn't revert. It also did not consider it a successful run for me, which was good, but it left the service trying to deploy the new definition even though it would never work.